README.md

```diff
@@ -6,3 +6,9 @@ XDF is a general-purpose container format for multi-channel time series data wit
 XDF is open source and designed to be a community project; that is, everyone is invited to contribute to the format [specification](https://github.com/sccn/xdf/wiki/Specifications). The core specification of the format is kept as simple as possible, but at the same time rich enough to support features such as time stamp corrections, boundary chunks for seeking, and data streams supporting many different data types (including int, float, double, and string).
 
 The XDF concept and beta version was developed by Christian Kothe and Clemens Brunner at the [Swartz Center for Computational Neuroscience (SCCN)](http://sccn.ucsd.edu/) at the [University of California San Diego, CA, USA](http://www.ucsd.edu/).
+
+# Repository Structure
+
+The XDF specification is outlined in the home repository at https://github.com/sccn/xdf . As of late april 2019, XDF readers, writers, and associated tools can be found in the https://github.com/xdf-modules organization.
+
+The most popular XDF importers (for Python, Matlab, and EEGLAB) are linked in the home repository as submodules. For this reason, if your intention is to download xdf and its implementations then you shouldn't use the green "Download" button in GitHub (which doesn't download submodules), and when you clone the repository please be sure to use the `--recursive` flag.
```

Comment:
The most popular XDF importers (for Python, Matlab, and EEGLAB) are linked in the home repository as submodules. For this reason, if your intention is to download XDF and its implementations then you shouldn't use the green "Download" button in GitHub (which doesn't download submodules), and when you clone the repository please be sure to use the `--recursive` flag.

If you are interested in a specific reader implementation (such as the Python reader `pyxdf`), you can directly clone a repository from the https://github.com/xdf-modules organization.

Reply:
Not including this last bit of information is also intentional. Advanced users will know this already. Novice users, however, are more likely to blame sccn for any trouble they encounter (whether it's from the parent repo or the subrepo) so I think sccn should maintain some level of control over which versions/commits of the submodules are linked from the parent repo.

My goal is to give the community maintainers as much power as they need to move forward without relying on SCCN personnel for maintenance, while still keeping SCCN in control of its own representation. I think this balance is necessary to keep the community healthy.

Reply:
OK, then just XDF instead of xdf.